src/app/_components/DistrictSearch/DistrictList.js

```diff
@@ -0,0 +1,56 @@
+'use client';
+
+import { Tick } from '@icons/index';
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+import 'swiper/css';
```

Comment:
You can create a component named Slider, which will import Swiper and swiper.css and reexport Swiper and SwiperSlide.

Reply:
Create a Slider component. Could you please take a look at it to see whether I understood your suggestion correctly?